src/title-page.md

```diff
@@ -2,14 +2,17 @@
 
 *由 Steve Klabnik 與 Carol Nichols，以及 Rust 社群的貢獻撰寫而成*
 
-此版本假設你使用的是 Rust 1.58（於 2022-01-13 發布）或更高的版本，並在所有專案中的 *Cargo.toml* 都有 `edition="2021"` 來使用 Rust 2021 版號。請查看[第一章的「安裝」段落][install]<!-- ignore -->來安裝或更新 Rust。
+此版本假設你使用的是 Rust 1.65（於 2022-11-03 發布）或更高的版本，並在所有專案中的 *Cargo.toml* 都有 `edition="2021"` 來使用 Rust 2021 版號。請查看[第一章的「安裝」段落][install]<!-- ignore -->來安裝或更新 Rust。
 
 本書的 HTML 格式可以在線上閱讀：[https://doc.rust-lang.org/stable/book/](https://doc.rust-lang.org/stable/book/)（[正體中文版](https://rust-lang.tw/book-tw/)）。而離線版則包含在 `rustup` 安裝的 Rust 中，輸入 `rustup docs --book` 就能開啟。
 
+社群中也有提供本書的各種[譯本]。
+
 本書也有由 [No Starch Press 出版平裝與電子版格式][nsprust]。
 
 [install]: ch01-01-installation.html
 [editions]: appendix-05-editions.html
 [nsprust]: https://nostarch.com/rust
+[translations]: appendix-06-translation.html
 
```

Comment:
Do you want to mention the experimental Rust book from Brown Univerisy here?
```suggestion

> **🚨 想要更有互動的學習體驗？來嘗試不同的 Rust Book，賣點有：隨堂測驗、重點提示、視覺化，還有更多**：<https://rust-book.cs.brown.edu>

```

Reply:
I'll add it.